src/mobster/cli.py

```diff
@@ -266,6 +266,12 @@ def generate_augment_oci_image_parser(subparsers: Any) -> None:
         type=Path,
         help="path to public key used to verify the image provenance",
     )
+    augment_oci_image_parser.add_argument(
+        "--concurrency",
+        type=parse_concurrency,
```

Comment:
Out of curiosity... if you define `type=int` does it also accept negative numbers?